03_module/Makefile

```diff
@@ -0,0 +1,9 @@
+obj-m += simple_module.o
+
+BUILD_KERNEL := ~/repos/busybox/_install/lib/modules/4.19.177/build
+KDIR = ~/repos/linux-stable
```

Comment:
Use `?=` assignment in the two rows above to allow build the module without changing of the makefile.